oat-sa/extension-tao-itemqti-pic
# Fix/NCC-209/Arrange PIC rendering calls

**Related to task:**  https://oat-sa.atlassian.net/browse/NCC-209
**Description:** There are two issues solves:
1. PIC rendering calls are disorderly. Student tools PICs (PIC1 and PIC2 on the image) depends on a student toolbar (PIC3), so the PIC3 should be rendered first. 
The issue is solved by arrange PIC rendering calls
<img width="410" alt="Screenshot 2020-05-29 at 09 45 53" src="https://user-images.githubusercontent.com/6339002/83235996-156dc000-a193-11ea-9812-95e59a7dc8b6.png">

2. By opening item in the preview mode PIC icons in tool bar are rendering to the authoring page.
The issue is solved by selecting toolbar component in scope of item. [Fix for "Document viewer" PIC](https://github.com/oat-sa/extension-tao-nccer/compare/develop...fix/NCC-209/render-PIC-icons-on-the-preview-page?expand=1)

**How to test:** 
- Create a new item
- On the widget panel under "Student Tools" select "Document Viewer" option
- "Document Viewer" icon should appear on item toolbar (issue 1)
- Save the item and open a preview mode 
- On preview page icons from "Student tools" should be visible (issue 2)

**Unit tests cli:** `npx grunt testall`
